exercises/custom-set/canonical-data.json

```diff
@@ -131,6 +131,26 @@
             "set2": [4, 1, 3]
           },
           "expected": false
+        },
+        {
+          "uuid": "0b95204d-93fa-4b60-9499-1e34f228c336",
+          "description": "set is a subset of set with duplicates",
```

Comment:
A set with with duplicates isn't technically a set 🤷

Reply:
Yes :sweat_smile: but it's shorter than "a set constructed from an array with duplicates" :joy:

Reply:
It is, but I actually like the second version better!

Reply:
OK, [chanded it](https://github.com/exercism/problem-specifications/pull/2324/commits/915bf818ab226d0e16c7452fd6981e8684f7fb08)!